tests/test_hep_bdFFT.py

```diff
@@ -115,42 +111,6 @@ def test_fft_from_FFT_ignores_context():
     assert '_fft' not in result
 
 
-def test_fft_from_FFT_does_not_require_s():
```

Comment:
I think you should preserve this test too. It's true that we no longer use `FFT__s`, but it was here because it documented a bug fix from which we should not regress.

Reply:
Same goes for `test_fft_from_FFT_percent_percent`: AFAIK it no longer happens in Legacy, but we should keep the test, otherwise people see the regex `FFT[^%][^%]` and have no idea why it's there.